docs/demo.rst

```diff
@@ -217,6 +217,156 @@ using:
+
+    ga4gh_repo add-referenceset registry.db /full/path/to/hs37d5.fa.gz -d “NCBI37 assembly of the human genome” --ncbiTaxonId 9606 --name NCBI37 --sourceUri "ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5.fa.gz"
+
+A number of optional command line flags have been added. We will be
+referring to the name of this reference set ``NCBI37`` when we later add
+the variant set.
+
+Add an ontology TODO
+--------------------
+
+Ontologies provide a source for parsing variant annotations, as well as
+organizing feature types into ontology terms. This is a custom format
+created for this server.
+
+.. code-block:: bash
+
+    ga4gh_repo add-ontology registry.db /full/path/to/sequence_ontology.txt
+
+Add sequence annotations
+------------------------
+
+The GENCODE Genes dataset provides annotations for features on the
+reference assembly. These features are made available to the server by
+preparing the GENCODE genes GFF3 into an SQLite database. The script
+used to generate this file is available in the ``scripts`` directory of
+the source. This can take some time. You can skip this step by
+downloading a sequence annotation set made using this method
+`here <https://ga4ghstore.blob.core.windows.net/testing/gencode_v24lift37.db>`__.
+
+.. code-block:: bash
+
+    wget ftp://ftp.sanger.ac.uk/pub/gencode/Gencode_human/release_24/GRCh37_mapping/gencode.v24lift37.annotation.gff3.gz
+    gunzip gencode.v24lift37.annotation.gff3.gz
+    python scripts/generate_gff3_db.py -i gencode.v24lift37.annotation.gff3 -o gencode.v24lift37.annotation.db
+
+Now that the feature set has been downloaded and prepared, it can be
+added to the registry using the following command. Notice we have told
+the registry to associate the reference set added above with these
+annotations.
+
+.. code-block:: bash
+
+    ga4gh_repo add-featureset registry.db 1kgenomes /full/path/to/gencode.v24lift37.annotation.db --referenceSetName NCBI37
+
+Add the Thousand Genomes VCFs
+-----------------------------
+
+The Thousand Genomes are publicly available on the EBI server. This
+command uses ``wget`` to download the "release" VCF to a directory named
+release.
+
+.. code-block:: bash
+
+    wget -m ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/ -nd -P release -l 1
+    rm release/ALL.wgs.phase3_shapeit2_mvncall_integrated_v5b.20130502.sites.vcf.gz
+
+These files are already compressed and indexed. For the server to make use
+of the files in this directory we must move the `wgs` file, since it covers 
+chromosomes that are represented elsewhere and overlapping VCF are not 
+currently supported. This file could be added as a separate variant set.
+
+We can now add thedirectory to the registry using the following command.
+Again, notice we have referred to the reference set by name.
+
+.. code-block:: bash
+
+    ga4gh_repo add-variantset registry.db 1kgenomes /full/path/to/vcf/ --name release --referenceSetName NCBI37
```

Comment:
/full/path/to/vcf/ is a bit confusing as it implies a single vcf. You should use the same directory as the command above to facilitate cut-n-paste.